<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeechCraft</class>
 <widget class="QMainWindow" name="LeechCraft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LeechCraft</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainresources.qrc">
    <normaloff>:/resources/images/leechcraft.svg</normaloff>:/resources/images/leechcraft.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="LeechCraft::SeparateTabWidget" name="MainTabWidget_" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActionAddTask_">
   <property name="text">
    <string>Add task</string>
   </property>
  </action>
  <action name="ActionSettings_">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="ActionQuit_">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="ActionIcon" stdset="0">
    <string>exit</string>
   </property>
  </action>
  <action name="ActionFullscreenMode_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen mode</string>
   </property>
   <property name="toolTip">
    <string>Fullscreen mode</string>
   </property>
  </action>
  <action name="ActionLogger_">
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>Log</string>
   </property>
  </action>
  <action name="ActionAboutLeechCraft_">
   <property name="text">
    <string>About LeechCraft...</string>
   </property>
  </action>
  <action name="ActionAboutQt_">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="ActionMenu_">
   <property name="icon">
    <iconset resource="mainresources.qrc">
     <normaloff>:/resources/images/leechcraft.svg</normaloff>:/resources/images/leechcraft.svg</iconset>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="ActionShowStatusBar_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show status bar</string>
   </property>
  </action>
  <action name="ActionCloseTab_">
   <property name="text">
    <string>Close tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="ActionNewTab_">
   <property name="text">
    <string>New tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="ActionShowToolBar_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show toolbar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::SeparateTabWidget</class>
   <extends>QWidget</extends>
   <header>separatetabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainresources.qrc"/>
 </resources>
 <connections/>
</ui>
